ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -783,13 +783,12 @@ <h3 class="title">
       <ng-container matColumnDef="spIndexOrder">
         <th class="table_header" mat-header-cell *matHeaderCellDef>Column Order</th>
         <td mat-cell *matCellDef="let element">
-          <div *ngIf="isIndexEditMode && element.get('spOrder').value">
+          <div *ngIf="isIndexEditMode && element.get('spColName').value">
```

Comment:
If I try to change the order of columns in indexes it reverts to original value

Reply:
Solved